<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYInterpolationCurveDockGeneralTab</class>
 <widget class="QWidget" name="XYInterpolationCurveDockGeneralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>953</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="2" colspan="2">
    <widget class="QComboBox" name="cbDataSourceType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="ResizableTextEdit" name="teComment"/>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lData">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lDataSourceType">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lDataSourceCurve">
     <property name="text">
      <string>Curve:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lXColumn">
     <property name="text">
      <string>X-Data:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lYColumn">
     <property name="text">
      <string>Y-Data:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lXRange">
     <property name="text">
      <string>X-Range:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QCheckBox" name="cbAutoRange">
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QFrame" name="frameRange">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lMax">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lMaxDateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lMinDateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lMin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="UTCDateTimeEdit" name="dateTimeEditMin">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="UTCDateTimeEdit" name="dateTimeEditMax">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leMin"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leMax"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lInterpolation">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Interpolation</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lType">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="KComboBox" name="cbType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lVariant">
     <property name="text">
      <string>Variant:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QComboBox" name="cbVariant"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lParameter">
     <property name="text">
      <string>Parameter:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lTension">
       <property name="text">
        <string>Tension:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NumberSpinBox" name="sbTension">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>Spinbox for numbers.</string>
       </property>
       <property name="feedback" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lContinuity">
       <property name="text">
        <string>Continuity:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NumberSpinBox" name="sbContinuity">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>Spinbox for numbers.</string>
       </property>
       <property name="feedback" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lBias">
       <property name="text">
        <string>Bias:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NumberSpinBox" name="sbBias">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>Spinbox for numbers.</string>
       </property>
       <property name="feedback" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lEval">
     <property name="text">
      <string>Evaluate:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="2">
    <widget class="QComboBox" name="cbEval"/>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lPoints">
     <property name="text">
      <string>Points:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDoubleSpinBox" name="sbPoints">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>number should be bigger than number of data points</string>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPointsMode">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="17" column="2">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>214</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="lResults">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Results:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="18" column="2" colspan="2">
    <widget class="QTextEdit" name="teResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="19" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="20" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>266</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="3">
    <widget class="QPushButton" name="pbRecalculate">
     <property name="text">
      <string>Recalculate</string>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <spacer name="verticalSpacerGeneral">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="23" column="0">
    <widget class="QCheckBox" name="chkVisible">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="QComboBox" name="cbPlotRanges"/>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="lPlotRange">
     <property name="text">
      <string>Plot Range:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>UTCDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>UTCDateTimeEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
